src/sph-partition.cc

```diff
@@ -125,12 +174,24 @@ void Worker::defineOptions(po::options_description & opts) {
          "to the output field name list if it isn't already included.")
         ("part.sub-chunk",
          po::value<std::string>()->default_value("subChunkId"),
-         "Sub-chunk ID output field name. This field field name is appended "
+         "Sub-chunk ID output field name. This field name is appended "
          "to the output field name list if it isn't already included.")
+        ("part.id",
+         po::value<std::string>(),
+         "Object ID input field name. This field name is optional. If it's provided "
+         "then the secondary index map file will be generated.")
         ("part.pos",
          po::value<std::string>(),
          "The partitioning longitude and latitude angle field names, "
-         "separated by a comma.");
+         "separated by a comma.")
+        ("part.id-url",
+         po::value<std::string>(),
+         "Universal resource locator for an existing secondary index.")
```

Comment:
"URL" is probably better than "Universal resource locator" in this case, for me it takes more mental power to parse and understand latter.